src/ert/_c_wrappers/job_queue/queue.py

```diff
@@ -449,30 +455,23 @@ async def execute_queue_via_websockets(  # pylint: disable=too-many-arguments
             headers["token"] = token
 
         try:
-            await JobQueue._publish_changes(
-                ens_id, self._differ.snapshot(), ws_uri, ssl_context, headers
-            )
-            while True:
-                self.launch_jobs(pool_sema)
-
-                await asyncio.sleep(1)
-
-                for func in evaluators:
-                    func()
-
+            # initial publish
+            async with connect(
+                ws_uri,
+                ssl=ssl_context,
+                extra_headers=headers,
+                open_timeout=60,
+                ping_timeout=60,
+                ping_interval=60,
+                close_timeout=60,
+            ) as websocket:
```

Comment:
wondering whether `websocket` is the best name here. What about being explicit `queue_connection` or something along those lines?

Reply:
indeed - websocket is a bit generic, I guess - or could you make more specific in which sense `websocket` is not the best name?

I'm having a hard time coming up with something better though :sweat_smile: 

I feel in the suggested `queue_connection`, the `queue` part is a bit redundant, as it's used within the `JobQueue` class - leaving just `connection, which is an alternative to `websocket`, but perhaps just as generic or bad.

Maybe it could be named according to what it connects to, like `evaluator_connection` - does that make more sense? it gives a coupling to the name of the ensemble evaluator though. 

mind also that the connection information is passed in as a parameter called `ws_uri`, which in itself is also a very generic name. perhaps it would be more consistent to either leave the contested name to be `websocket`, or also change e.g. `ws_uri` to be more descriptive?

naming, eh? one of the hardest problems...

Reply:
1. What about `ee_connection` then? Since ee is used in our code base to refer to ensemble_evaluator.
2. ws_uri -> `ee_uri` ?

Reply:
sure! let me fix it now

Reply:
done!